src/metabase/driver/mysql.clj

```diff
@@ -65,7 +65,8 @@
       kdb/mysql
       ;; 0000-00-00 dates are valid in MySQL, but JDBC barfs when queries return them because java.sql.Date doesn't allow it.
       ;; Add a param to the end of the connection string that tells MySQL to convert 0000-00-00 dates to NULL when returning them.
-      (update :subname (u/rpartial str "?zeroDateTimeBehavior=convertToNull"))))
+      ;; Also add params to force UTF-8 encoding of results
+      (update :subname (u/rpartial str "?zeroDateTimeBehavior=convertToNull&useUnicode=true&characterEncoding=UTF8&characterSetResults=UTF8"))))
```

Comment:
Keep in mind that our code, including API responses, pretty much assumes everything is UTF-8. Our codebase is UTF-8 encoded. Browsers and Excel handle UTF-8 just fine. If we let someone configure their DB encoding we'll have to adjust the `Content-Type` header too and fix our assumption everywhere else in the codebase. 

[Even Java people will tell you UTF-8 everywhere is a best practice](http://www.javapractices.com/topic/TopicAction.do?Id=206). Since all of our code already assumes that is the case I think it makes ¢¢¢ to just to tell databases to return results as UTF-8 rather than making work for ourselves thinking about a hypothetical scenario where one rando wants to use [circa-1985 IBM Extended ASCII](https://en.wikipedia.org/wiki/Extended_ASCII) so he can do line art.